1-js/06-advanced-functions/12-arrow-functions/article.md

```diff
@@ -61,28 +61,27 @@ let group = {
 group.showList();
 ```
 
-The error occurs because `forEach` runs functions with `this=undefined` by default, so the attempt to access `undefined.title` is made.
+এক্ষেত্রে `forEach` এর ফাংশনটির ডিফল্ট কন্টেক্স হত `this=undefined`, সুতরাং এটি প্রপার্টিকে এভাবে অ্যাক্সেস করতে চাইত `undefined.title`।
 
-That doesn't affect arrow functions, because they just don't have `this`.
+যেহেতু অ্যারো ফাংশনের `this` নেই, তাই এটি কোন এরর রিটার্ন করবে না।
 
-```warn header="Arrow functions can't run with `new`"
-Not having `this` naturally means another limitation: arrow functions can't be used as constructors. They can't be called with `new`.
+```warn header="`new` এ সাথে অ্যারো ফাংশন রান করে না"
```

Comment:
Please change: `new` এ সাথে অ্যারো ফাংশন রান করে না"
To : `new` এর সাথে অ্যারো ফাংশন রান করে না"